### PR TITLE
feat: Rework settings page

### PR DESCRIPTION
**Pull request by ptbrowne**
Use the same kind of card as the category alert edit modal.

### CODE REVIEW

File: src/locales/fr.json [hunk 1]
```diff
@@ -404,13 +404,19 @@
...
       "settingTitle": "Solde bas",
+      "fieldLabels": {
+        "value": "Me prévenir quand mon solde passe sous"
+      },
       "email": {
         "innerTitle": "Notification de solde bas",
         "access_accounts": "Accéder à mes comptes"
       },
-      "description": "Vous recevrez une notification, si votre solde est inférieur à",
+      "description": "Vous recevrez une notification, si votre solde est inférieur à *%{value}*€",
```
Comment (drazik) [inline src/locales/fr.json]: There's no comma in the english version, and I think it's better to read without it.

File: src/locales/fr.json [hunk 2]
```diff
@@ -425,10 +431,13 @@
     },
     "if_transaction_greater": {
       "settingTitle": "Mouvements",
-      "description": "Vous recevrez une notification, si un mouvement est supérieur à",
+      "description": "Vous recevrez une notification, si un mouvement est supérieur à *%{value}*€",
```
Comment (drazik) [inline src/locales/fr.json]: Same here for the comma

File: src/components/EditionModal/EditionModal.jsx [hunk 3]
```diff
@@ -88,20 +94,19 @@ const InfoSlide = ({
...
             value={fieldSpec.getValue(doc)}
-            onClick={
-              fieldSpec.immediate ? null : () => onRequestChooseField(fieldName)
-            }
+            onClick={hasSelector ? () => onRequestChooseField(fieldName) : null}
             onChange={
-              fieldSpec.immediate
-                ? value => {
+              hasSelector
+                ? null
```
Comment (drazik) [inline src/components/EditionModal/EditionModal.jsx]: Don't we want to pass `undefined`, to act as if the prop wasn't passed at all? (for example if there is a default prop)

File: src/components/SettingCard/index.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+      {...mergeProps(props, {
+        className: cx(
+          styles.SettingCard,
+          dashed && styles['SettingCard--dashed'],
+          clickable && styles['SettingCard--clickable']
+        )
+      })}
+    />
+  )
+}
```
[the remark this answers is not in the document]
Comment (ptbrowne) [reply src/components/SettingCard/index.jsx]: No, you're right it was too complex.

File: src/components/EditionModal/EditionModal.jsx [hunk 5]
```diff
@@ -22,6 +23,16 @@ export const CHOOSING_TYPES = {
...
 
+const BackArrow = ({ onClick }) => (
+  <span className="u-mr-half">
+    <Icon
+      style={{ cursor: 'pointer', color: 'var(--coolGrey)' }}
+      icon="previous"
+      onClick={onClick}
+    />
+  </span>
+)
```
[the remark this answers is not in the document]
Comment (ptbrowne) [reply src/components/EditionModal/EditionModal.jsx]: Done :)

**Pull request merged**